src/util/chaintype.cpp

```diff
@@ -15,6 +15,8 @@ std::string ChainTypeToString(ChainType chain)
         return "main";
     case ChainType::TESTNET:
         return "test";
+    case ChainType::TESTNET4:
+        return "testnet4";
```

Comment:
Can this be `test4`? The inconsistency looks ridiculous. Although `signet` has a similar problem already.

Reply:
I'm not so sure. I think anywhere we print this string `testnet4` it is nicer because it's closer to what users actually say. I also interpret the choice that was made for `signet` to be what we want to do going forward, since it was the latest addition on the list. So I tent to leave this as is unless there are other arguments than the inconsistency.

Reply:
Well, I guess `test` should've been named `testnet3` from the start. Maybe this is an improvement but I guess it will *also* confuse people who are used to the old naming or think they can just change `test` to `test4` in their configs. Maybe this should be explicitly called out in release post or something.

Reply:
Ok, I will keep in mind making it explicit in the release notes/docs.